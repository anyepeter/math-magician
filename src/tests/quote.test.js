import { render } from '@testing-library/react';
import Quotes from '../pages/quote';

describe('display the quotes page', () => {
  it('check the contents of the quote page', () => {
    const { container } = render(
      <Quotes />,
    );
    expect(container).toMatchSnapshot();
  });
});
