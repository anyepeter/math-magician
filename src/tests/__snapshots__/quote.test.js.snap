// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`display the  check the contents of the home page 1`] = `
<div>
  <div
    class="text-container"
  >
    <p>
      Mathematics is not about numbers, equations, computations or algorithms; it is about understanding - William Paul Thurston.
    </p>
  </div>
</div>
`;

exports[`display the home page check the contents of the home page 1`] = `
<div>
  <div
    class="text-container"
  >
    <p>
      Mathematics is not about numbers, equations, computations or algorithms; it is about understanding - William Paul Thurston.
    </p>
  </div>
</div>
`;

exports[`display the quotes page check the contents of the home page 1`] = `
<div>
  <div
    class="text-container"
  >
    <p>
      Mathematics is not about numbers, equations, computations or algorithms; it is about understanding - William Paul Thurston.
    </p>
  </div>
</div>
`;

exports[`display the quotes page check the contents of the quote page 1`] = `
<div>
  <div
    class="text-container"
  >
    <p>
      Mathematics is not about numbers, equations, computations or algorithms; it is about understanding - William Paul Thurston.
    </p>
  </div>
</div>
`;

exports[`display the quotes page check the contents of the quto page 1`] = `
<div>
  <div
    class="text-container"
  >
    <p>
      Mathematics is not about numbers, equations, computations or algorithms; it is about understanding - William Paul Thurston.
    </p>
  </div>
</div>
`;
